### DISCUSSION TITLE
"NeighborLoader" does not seem to be affected by seed.

### DISCUSSION THREAD

**Discussion by lck-handsome** [pyg-team/pytorch_geometric#9432] [Q&A]
"NeighborLoader" does not seem to be affected by seed.
I set "torch_geometric.seed_everything(seed)",
When I use num_neighbors=[1,2,4] it is obvious that the size of the sampled subgraphs is not consistent. I wonder if I have set something wrong and how do I control the sampling seeds

**Comment by rusty1s** [comment]
Do you have `pyg-lib` installed? Seeds should definitely be respected there.
